packages/fxa-content-server/app/scripts/views/password_strength/password_strength_balloon.js

```diff
@@ -33,9 +29,8 @@ class PasswordStrengthBalloonView extends BaseView {
   template = Template;
 
   initialize(config = {}) {
-    this.delayBeforeHideMS = config.delayBeforeHideMS || DELAY_BEFORE_HIDE_MS;
-
     this.listenTo(this.model, 'change:password', this.update);
+    this.listenTo(this.model, 'change:inputFocus', this.shouldHide);
```

Comment:
I needed a way to consume an event in `password_strength_balloon.js` that's set in `password_with_strength_balloon.js` because I haven't seen where we set `events` in a class and I thought it made for better separation of concerns anyway. If setting this on `this.model` isn't a good approach I'm happy to change it in favor of a better suggestion.